core/storage/base_model/gae_models.py

```diff
@@ -1259,6 +1259,49 @@ def put(self, *args: Any, **kwargs: Any) -> None:
             'The put() method is missing from the '
             'derived class. It should be implemented in the derived class.')
 
+    def get_models_to_put_values(
+        self,
+        committer_id: str,
+        commit_message: str,
+        commit_cmds: List[Dict[str, Union[str, int]]]
+    ) -> List[BaseModel]:
+        """Returns the models which should be put.
+
+        Args:
+            committer_id: str. The user_id of the user who committed the change.
+            commit_message: str. The commit description message.
+            commit_cmds: list(dict). A list of commands, describing changes
+                made in this model, should give sufficient information to
+                reconstruct the commit. Dict always contains:
+                    cmd: str. Unique command.
+                And then additional arguments for that command. For example:
+
+                {'cmd': 'AUTO_revert_version_number'
+                 'version_number': 4}
+
+        Returns:
+            list(BaseModel). The models which should be put.
+        """
+        commit_type = (
+            feconf.COMMIT_TYPE_CREATE
+            if self.version == 0
+            else feconf.COMMIT_TYPE_EDIT
+        )
+
+        models_to_put = self.compute_models_to_commit(
+                committer_id,
+                commit_type,
+                commit_message,
+                commit_cmds,
+                self._prepare_additional_models()
+            )
```

Comment:
This variable name sounds like a list, maybe you should do the .values() call here so that models_to_put is a list.